### PR TITLE
GH Migration

### PR DESCRIPTION
**Pull request by anandrachit**

### CODE REVIEW

File: app/bundle.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use strict";
```
Comment (ryan-ward-samsung) [inline app/bundle.js:1]: Please delete this file.

File: app/src/polyfill.js [hunk 2]
```diff
@@ -1 +1 @@
-import 'babel-polyfill';
```
Comment (ryan-ward-samsung) [inline app/src/polyfill.js:1]: Since we don't need this file, we should just delete it.

File: app/src/utils/health-check.util.js [hunk 3]
```diff
@@ -4,6 +4,8 @@ import path from 'path';
 import { logger, redis } from './coreUtilsManager';
 import { modelPodcast } from './databaseUtils';
 
+// #TODO: Change modelPodcast to modelMachines here
+
 export class HealthCheckUtil {
   static allowSelfHeal = true;
```
Comment (ryan-ward-samsung) [inline app/src/utils/health-check.util.js:10]: Can we go ahead and do a simple DB operation for healthcheck here, so we have a working healthcheck util?

**Review by ryan-ward-samsung — CHANGES_REQUESTED** [review]
Over looks good. Just a couple of things to remove and we have a good baseline.

File: app/src/polyfill.js (hunk 2 above)
Comment (anandrachit) [reply app/src/polyfill.js:1]: Done.

File: app/bundle.js (hunk 1 above)
Comment (anandrachit) [reply app/bundle.js:1]: Done!

File: app/src/utils/health-check.util.js (hunk 3 above)
Comment (anandrachit) [reply app/src/utils/health-check.util.js:10]: Will not be able to do it in this branch as the DB has not been setup. Will do it in the branch we have for the POST machines API.

Comment (ryan-ward-samsung) [reply app/src/utils/health-check.util.js:10]: Okay, no worries. So long as we don't forget.

**Comment by ryan-ward-samsung** [comment]
You're good to merge.

**Review by ryan-ward-samsung — APPROVED** [review]

**Pull request merged**